blocks/library/button/index.js

```diff
@@ -67,18 +65,12 @@ class ButtonBlock extends Component {
 			text,
 			url,
 			title,
-			align,
 			color,
 			textColor,
 			clear,
 		} = attributes;
 
 		return [
-			isSelected && (
-				<BlockControls key="controls">
-					<BlockAlignmentToolbar value={ align } onChange={ this.updateAlignment } />
-				</BlockControls>
-			),
 			<span key="button" className={ className } title={ title } ref={ this.bindRef }>
```

Comment:
And this will no longer need `key` when not returned as part of an array.

Reply:
See my previous comment.